Setup.sh

```diff
@@ -4,10 +4,13 @@ set -e
 
 TEMPO_ROOT=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
+cd $TEMPO_ROOT
```

Comment:
This guarantees that the `git rev-parse --git-common-dir` command below is always run from Tempo, not from wherever the user ran the script from.